docs/Lua API.rst

```diff
@@ -1188,6 +1188,244 @@ Job module
 Units module
 ------------
 
+* ``dfhack.units.isUnitInBox(unit,x1,y1,z1,x2,y2,z2)``
```

Comment:
getUnitsInBox also needs documenting, I think

Reply:
It is, ctrl+f
I put all the is* functions at the top. I think get in box is the first one under